instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/conftest.py

```diff
@@ -52,6 +69,62 @@ def fixture_event_logger_provider(log_exporter):
     return event_logger_provider
 
 
+@pytest.fixture(scope="function", name="meter_provider")
+def fixture_meter_provider(metric_reader):
+    token_usage_histogram_view = View(
```

Comment:
Am not sure adding the views is useful since it's outside of the instrumentation concern, maybe wait for advisory support to get in instead?